account-kit/smart-contracts/src/light-account/accounts/account.ts

```diff
@@ -54,20 +45,21 @@ export type LightAccount<
 export type CreateLightAccountParams<
   TTransport extends Transport = Transport,
   TSigner extends SmartAccountSigner = SmartAccountSigner,
-  TLightAccountVersion extends GetLightAccountVersion<"LightAccount"> = GetLightAccountVersion<"LightAccount">,
-  TEntryPointVersion extends GetEntryPointForLightAccountVersion<
-    "LightAccount",
-    TLightAccountVersion
-  > = GetEntryPointForLightAccountVersion<"LightAccount", TLightAccountVersion>
+  TLightAccountVersion extends LightAccountVersion<"LightAccount"> = LightAccountVersion<"LightAccount">,
+  TEntryPointVersion extends LightAccountEntryPointVersion<"LightAccount"> = LightAccountEntryPointVersion<"LightAccount">
```

Comment:
Just curious: why does this type still need a `TEntryPointVersion` parameter rather than inferring the entry point version from `TLightAccountVersion` like in the other places? Likewise for `CreateMultiOwnerLightAccountParams` later.

Reply:
I think I ran into something by not having it generic here... but can't remember what it was. lemme try and remove this all together and infer it from the version